theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/CurveSegmentEditor.tsx

```diff
@@ -16,40 +17,43 @@ const PATTERN_DOT_SIZE = 0.01
 const PATTERN_DOT_COUNT = 8
 const PATTERN_GRID_SIZE = (1 - PATTERN_DOT_SIZE) / (PATTERN_DOT_COUNT - 1)
 
-const CURVE_START_OVERSHOOT_COLOR = 'rgb(64, 170, 164)'
-const CURVE_START_COLOR = 'rgb(64, 170, 164)'
-const CURVE_MID_START_COLOR = 'rgb(64, 170, 164)'
-const CURVE_MID_COLOR = 'rgb(64, 170, 164)'
-const CURVE_MID_END_COLOR = 'rgb(64, 170, 164)'
-const CURVE_END_COLOR = 'rgb(64, 170, 164)'
-const CURVE_END_OVERSHOOT_COLOR = 'rgb(64, 170, 164)'
+const CURVE_START_OVERSHOOT_COLOR = '#3EAAA4'
+const CURVE_START_COLOR = '#3EAAA4'
+const CURVE_MID_START_COLOR = '#3EAAA4'
+const CURVE_MID_COLOR = '#3EAAA4'
+const CURVE_MID_END_COLOR = '#3EAAA4'
+const CURVE_END_COLOR = '#3EAAA4'
+const CURVE_END_OVERSHOOT_COLOR = '#3EAAA4'
 
 const CONTROL_COLOR = '#B3B3B3'
 
 const Circle = styled.circle`
   stroke-width: 0.1px;
   vector-effect: non-scaling-stroke;
-  r: 0.05px;
+  r: 0.04px;
   pointer-events: none;
+  transition: r 0.15s;
+  fill: #3eaaa4;
 `
 
 const HitZone = styled.circle`
   stroke-width: 0.1px;
   vector-effect: non-scaling-stroke;
-  r: 0.1px;
+  r: 0.09px;
   cursor: move;
   ${pointerEventsAutoInNormalMode};
   &:hover {
+    opacity: 0.4;
   }
   &:hover + ${Circle} {
-    r: 0.1px;
+    fill: #67dfd8;
```

Comment:
Nit: Awkward constant color defined separate from all the other color constants.